string name_space
string object_type
---
string[] location